pages/random/[url].tsx

```diff
@@ -24,11 +26,13 @@ const random = (): JSX.Element => {
   const [isTimerOver, setIsTimerOver] = useState(false)
   let timer: NodeJS.Timer
 
-  const handleStartVideo = () => {
-    if (isTimerOver) {
-      setIsVideoLoading(true)
-    } else {
-      alert('지금은 선물을 받을 수 없어요!')
+  const handleSliderChange = (event: Event, newValue: number | number[]) => {
+    if (newValue === 100) {
+      if (isTimerOver) {
+        setIsVideoLoading(true)
+      } else {
+        alert('djskdjskd')
+      }
```

Comment:
if(newValue === 100 && isTimerOver)로 체크하는 것이 어떨까요?
두번째 if의 else를 보면 의미없는 코드로 보여져서 위와 같은 방법이 어떨까 싶네요!

Reply:
```suggestion
  const handleSliderChange = (event: Event, newValue: number | number[]) => {
    if (newValue === 100 && isTimerOver ) {
        setIsVideoLoading(true)
      }
```

else 조건을 사용할 필요가 없다고 판단하여 위와 같이 수정하였습니다.